documentation/guides/release-and-deploy.md

```diff
@@ -49,3 +49,17 @@ git push origin master --follow-tags
 - Log in to [Shipit](https://shipit.shopify.io/shopify/quilt/production)
 
 - When CI is 🍏 on the commit titled `Publish`, press `Deploy` to update packages on npm.
+
+# Beta releases for 🎩ing changes?
+
+### Step 1 - publish a beta(s)
+
+- In your branch, run `yarn run release-beta`. Lerna will launch it's CLI to select a version for the changed packages. Select the `Custom` option and enter a version with an appended `-beta.X` (eg. `0.29.10-my-feature-beta.1`)
+- Push your branch to Github with the newly created tags using `git push origin <branch> --follow-tags`
+- Create a temporary stack in Shipit that points to your beta branch
```

Comment:
This step could be more explicit

![image](https://user-images.githubusercontent.com/6950534/74468254-87323000-4e68-11ea-8a68-c41d0a06f6f5.png)

A not about replacing the `environment` with `beta-test-<branch>` would be good.